internal/actions/release.go

```diff
@@ -25,30 +26,64 @@ func Release() error {
 		return err
 	}
 
-	files, err := g.GetCommitedFiles()
-	if err != nil {
-		fmt.Printf("Failed to get commited files: %s\n", err.Error())
+	dir := "."
+	usingReleasesMd := false
+	// TODO: Maybe instead check for workflow dispatch event
```

Comment:
Could it make sense to provide workflow dispatch event without a particular target?

Reply:
Not at this point. Publishing files will always be coded with a specific target now